src/constants/index.js

```diff
@@ -1,8 +1,22 @@
 "use strict"
 
-export const statuses = [ "DENY", "ALLOW", "DISMISS" ]
-export const statusDeny = "DENY"
-export const statusAllow = "ALLOW"
-export const statusDismiss = "DISMISS"
+export const STATUS_DENY = "DENY"
+export const STATUS_ALLOW = "ALLOW"
+export const STATUS_DISMISS = "DISMISS"
```

Comment:
When this state can occur? Can we keep the domain boolean (ALLOW/DENY) without introducing a third state?

Reply:
what would be the initial value? we cannot use DENY or ALLOW